packages/azure-http-specs/specs/azure/encode/duration/main.tsp

```diff
@@ -0,0 +1,35 @@
+import "@typespec/http";
+import "@typespec/spector";
+import "@azure-tools/typespec-client-generator-core";
+
+using TypeSpec.Http;
+using global.Azure.ClientGenerator.Core;
+using Spector;
+
+@doc("Test for azure related encode decorator.")
+@scenarioService("/azure/encode/duration")
+@scenario
+@scenarioDoc("""
+  Test case for azure specific encoding. SDK should generate correct serialization format according to the set encoding.
+  """)
```

Comment:
Should you put the scenario/scenarioDoc on the operation, instead of the whole file?